app/uk/gov/hmrc/gform/restapi/RestApiController.scala

```diff
@@ -14,23 +14,36 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.gform.config
+package uk.gov.hmrc.gform.restapi
```

Comment:
Idea was to have common controller for all functionality which is only in backend. At the moment we expose api for: 
1. exposed-config
2. delete-file 

Note that controller does simply calling `GformConnector` (no logic here at all). This is why I wanted to group such calls in one group (package/controller). Later here would land all other functionalities exposed for 3rd party developers.

I hope this brings some more light into it.